jsf-ri/src/main/java/com/sun/faces/context/flash/ELFlash.java

```diff
@@ -744,6 +744,8 @@ public void doLastPhaseActions(FacesContext context, boolean outgoingResponseIsR
 
 
         setCookie(context, flashManager, flashManager.encode(), false);
+        
+        System.out.println(flashManager.innerMap.size());

```

Comment:
@spyrkob it looks like this System.out needs to be removed?

Reply:
Yes, sorry, I added it to verify last change and forgot to remove. I'll update the PR tomorrow evening or Sunday morning